.github/workflows/publish.yml

```diff
@@ -57,28 +102,88 @@ jobs:
+          mv ${{ env.PRODUCTION_TARGET }}/deb/subspace-desktop-opencl_${{ steps.extract_version.outputs.version }}_amd64.deb executables/subspace-desktop-opencl_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.deb
+        if: runner.os == 'Linux'
+
+      - name: Prepare installers for uploading (macOS intel)
+        run: |
+          mkdir executables
+          mv ${{ env.PRODUCTION_TARGET }}/dmg/subspace-desktop_${{ steps.extract_version.outputs.version }}_x64.dmg executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.dmg
+          # Zip it so that signature is not lost
+          ditto -c -k --rsrc executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.dmg executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.zip
+          rm executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.dmg
+        if: matrix.build.target == 'x86_64-apple-darwin'
+
+      - name: Prepare installers for uploading (macOS arm)
+        run: |
+          mkdir executables
+          mv ${{ env.PRODUCTION_TARGET }}/dmg/subspace-desktop_${{ steps.extract_version.outputs.version }}_aarch64.dmg executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.dmg
+          # Zip it so that signature is not lost
+          ditto -c -k --rsrc executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.dmg executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.zip
+          rm executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.dmg
+        if: matrix.build.target == 'aarch64-apple-darwin'
+
+      - name: Prepare installers for uploading (Windows)
+        run: |
+          mkdir executables
+          move ${{ env.PRODUCTION_TARGET }}/msi/subspace-desktop_${{ steps.extract_version.outputs.version }}_x64_en-US.msi executables/subspace-desktop_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.msi
+          move ${{ env.PRODUCTION_TARGET }}/msi/subspace-desktop-opencl_${{ steps.extract_version.outputs.version }}_x64_en-US.msi executables/subspace-desktop-opencl_${{ steps.extract_version.outputs.version }}_${{ matrix.build.suffix }}.msi
+        if: runner.os == 'Windows'
+
+      - name: Upload executables to artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: executables-${{ matrix.build.suffix }}
+          path: |
+            executables/*
+          if-no-files-found: error
+
+      - name: Upload executables to assets
+        uses: alexellis/upload-assets@0.3.0
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
should this also be `secrets.GITHUB_TOKEN`? If so, perhaps we should standardize the usage (either this way or the other)

https://github.com/subspace/subspace-desktop/pull/304/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7L70

Reply:
yeah it is probably `secrets.GITHUB_TOKEN`, let me push that change